package.json

```diff
@@ -40,8 +40,7 @@
     "core-js": "3.46.0",
     "debug": "4.4.3",
     "puppeteer-core": "^24.24.1",
-    "yargs": "18.0.0",
-    "zod": "^3.25.76"
```

Comment:
We should also explicitly add it to devDependecies and not rely on anything pulling it in for us